### DISCUSSION TITLE
MySql settings

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#389] [Feature Requests]
Hello,
I have asked serveral times on spigot discussion but still havent seen an answer.
So i ask it here: how can i make my CMI with MySql but only sync some stuff.
Nickname, Rank, mails, but not theire money, inventorys gamemodes etc.
In some other plugins i know you can change the names of the tables where the info gets saved into and that way you can make it to save it in different tables but for the stuff you want not to sync.

**Comment by LogGits** [comment]
As far as I am aware this is not possible. Thought it does seem like a useful feature.

**Comment by ghost** [comment]
Yes this is possible have a look at the litebans settings file for example. That also does it that exact way. Now we are stuck with cmi to sync all or nothing so basically i don't see the usefull part of it atm. With that it would be excellent. Sync what you want with the servers you want. 
LINE: 147 of file: https://hastebin.com/ocosaserin.xml
That is the litebans webinterfact settings file where you can fill in the database and then the table that is used to store what. So you can say bans under server1_bans for example and kicks under kicks and set that in all servers the same and the kicks will sync around. When you use this directly under bungeecord I dont see this option, but before when I used it under Spigot I had these options in my Minecrafr plugin folder there too and that way they do it and would be amazing to do it in cmi that way too.

**Comment by Fabrimat** [comment]
That would be a very useful feature

**Comment by LogGits** [comment]
I was referring to it not being possible with cmi currently. It is still a great feature and will add an extra level of functionality.
